### ISSUE TITLE
tmp_smiles.txt&smiles2graph_demo.py 

### ISSUE THREAD

**Issue by hzau96yhz** [UCSD-AI4H/drugchat#13]
The "tmp_smiles.txt" file in dataset fold is not provided

**Comment by youweiliang** [comment]
The dataset/tmp_smiles.txt is automatically generated by smiles2graph_demo.py after you run it.

**Comment by hzau96yhz** [comment]
![QQ图片20230916193856](https://github.com/UCSD-AI4H/drugchat/assets/53745234/06edc706-5af2-4d3b-8da9-4bd6f74433a5)
Hello！I ran the smiles2graph.py file in the rdkit environment, but the program kept running, as shown in the image above. According to the code in the smiles2graph.py file, the tem_smiles.pkl file is also not generated in the dataset folder. Do you know what's going on?

**Comment by hzau96yhz** [comment]
![QQ图片20230916193856](https://github.com/UCSD-AI4H/drugchat/assets/53745234/5a9624e0-f608-44ae-a045-f39e98ca3daa)

Hello！I ran the smiles2graph.py file in the rdkit environment, but the program kept running, as shown in the image above. According to the code in the smiles2graph.py file, the tem_smiles.pkl file is also not generated in the dataset folder. Do you know what's going on?

**Comment by hzau96yhz** [comment]
![QQ图片20230916193856](https://github.com/UCSD-AI4H/drugchat/assets/53745234/009f47b2-41bd-4c5f-bb98-751649fe44e4)

Hello！I ran the smiles2graph_demo.py file in the rdkit environment, but the program kept running, as shown in the image above. According to the code in the smiles2graph_demo.py file, the tem_smiles.pkl file is also not generated in the dataset folder. Do you know what's going on?

**Comment by hzau96yhz** [comment]
@youweiliang Hi，The same problem I mentioned above, if I run smiles2graph_demo.py, and it keeps running, how do I switch to drugchat? It feels like there's something wrong here.

**Comment by youweiliang** [comment]
You shouldn't switch to drugchat. You should start another terminal (or simply a Unix shell) to activate the drugchat environment and run the drugchat model. And importantly, make sure the two processes (the smiles2graph_demo.py and the drugchat model) can access the same file system so that they can exchange information via the txt or pkl files you mentioned. 

You seemed to use a docker container? Please note two Docker containers may not share the file systems even running on the same machine.

**Comment by hzau96yhz** [comment]
@youweiliang Thank you very much for your answer! You are absolutely right that I rented autodl to run the drugchat project and opened two terminals on one server to run files in two different environments. As you said, files cannot be shared in this case, do you have a good solution?

**Comment by hzau96yhz** [comment]
Hello, I did another experiment, I rented a server, and opened a terminal on the server. I first used the `nohup python -u smiles2graph_demo.py > output.txt 2>&1 &` command in rdkit to execute the `smiles2graph_demo.py` program in the background, then switched to the drugchat environment to run the bash file. However, the result of the reproduction still does not generate the tmp_smiles.txt or pkl file.
In the front end display, enter the SMILES string, and no files will be updated in the dataset directory.
